### PR TITLE
Remove all unwraps, expects and panics

### PR DESCRIPTION
**Pull request by lionel-faber**

### CODE REVIEW

File: src/keys/signature.rs [hunk 1]
```diff
@@ -78,24 +75,24 @@ impl From<(usize, threshold_crypto::SignatureShare)> for Signature {
     }
 }
 
-#[allow(clippy::derive_hash_xor_eq)]
-impl Hash for Signature {
-    fn hash<H: Hasher>(&self, state: &mut H) {
-        utils::serialise(&self).hash(state)
-    }
-}
+// #[allow(clippy::derive_hash_xor_eq)]
```
Comment (joshuef) [inline src/keys/signature.rs]: lets remove these

File: src/keys/signature.rs [hunk 2]
```diff
@@ -78,24 +75,24 @@ impl From<(usize, threshold_crypto::SignatureShare)> for Signature {
...
+//         utils::serialise(&self).unwrap_or_default().cmp(&utils::serialise(other))
+//     }
+// }
 
-impl PartialOrd for Signature {
-    fn partial_cmp(&self, other: &Signature) -> Option<Ordering> {
-        Some(self.cmp(other))
-    }
-}
+// impl PartialOrd for Signature {
```
Comment (joshuef) [inline src/keys/signature.rs]: and these

File: src/messaging/sender.rs [hunk 3]
```diff
@@ -94,12 +91,12 @@ pub enum EntitySignature {
     Section(BlsSignature),
 }
 
-#[allow(clippy::derive_hash_xor_eq)]
-impl Hash for EntitySignature {
-    fn hash<H: Hasher>(&self, state: &mut H) {
-        utils::serialise(&self).hash(state)
-    }
-}
+// #[allow(clippy::derive_hash_xor_eq)]
```
Comment (joshuef) [inline src/messaging/sender.rs]: for removal

File: src/messaging/sender.rs [hunk 4]
```diff
@@ -310,9 +307,9 @@ impl Entity {
     }
 }
 
-#[allow(clippy::derive_hash_xor_eq)]
-impl Hash for Entity {
-    fn hash<H: Hasher>(&self, state: &mut H) {
-        utils::serialise(&self).hash(state)
-    }
-}
+// #[allow(clippy::derive_hash_xor_eq)]
```
Comment (joshuef) [inline src/messaging/sender.rs]: for removal

**Review by joshuef — CHANGES_REQUESTED** [review]
just the commented code to be removed and we're good 👍

**Review by joshuef — APPROVED** [review]

**Comment by lionel-faber** [comment]
Merging this since the CI had passed already and the only change after that was removal of commented code.

**Pull request merged**